packages/joyid/src/constants.ts

```diff
@@ -0,0 +1,82 @@
+// https://github.com/duanyytop/joyid-sdk-js/blob/4629f5158a5f54f00d765832d2d78fb20bb7c64b/src/constants/index.ts
+import { CellDep, Script } from "@ckb-lumos/base";
+
+const TestnetInfo = {
+  JoyIDLockScript: {
+    codeHash:
+      "0xd23761b364210735c19c60561d213fb3beae2fd6172743719eff6920e020baac",
+    hashType: "type",
+    args: "",
+  } as Script,
+
+  JoyIDLockDep: {
+    outPoint: {
+      txHash:
+        "0x4dcf3f3b09efac8995d6cbee87c5345e812d310094651e0c3d9a730f32dc9263",
+      index: "0x0",
+    },
+    depType: "depGroup",
+  } as CellDep,
+
+  CotaTypeScript: {
+    codeHash:
+      "0x89cd8003a0eaf8e65e0c31525b7d1d5c1becefd2ea75bb4cff87810ae37764d8",
+    hashType: "type",
+    args: "0x",
+  } as Script,
+
+  CotaTypeDep: {
+    outPoint: {
+      txHash:
+        "0x636a786001f87cb615acfcf408be0f9a1f077001f0bbc75ca54eadfe7e221713",
+      index: "0x0",
+    },
+    depType: "depGroup",
+  } as CellDep,
+};
+
+const MainnetInfo = {
```

Comment:
I see that these codes are not being used, and directly import from `joyid/ckb`, what's this here for?

Reply:
the same reason here https://github.com/ckb-js/lumos/pull/653#discussion_r1529582163